bigbluebutton-html5/imports/ui/stylesheets/variables/focus.scss

```diff
@@ -1,10 +1,19 @@
+// border
 $focus-border-size: 2px;
 $focus-border-offset: -4px;
+
+// colours
+$color-white: #FFF;
+$user-list-bg: #F3F6F9;
 $color-primary: #0F70D7 !default;
 $color-primary-input-shadow: lighten($color-primary, 35%);
-$user-list-bg: #F3F6F9;
-$color-white: #FFF;
 $list-item-bg-hover: darken($user-list-bg, 7%);
+$color-list-item-hover-bg: #e4ecf2;
+$color-list-item-focus-border: #92bcea;
+
+// padding 
+$sm-padding-y: .25rem;
+$lg-padding-y: .5rem;
```

Comment:
By changing global variables and importing the file on the `_all` you are actually overwriting the default values.
You should import the needed variables here and use them instead of duplicating it.